src/frontend/admin/src/components/templates/courses-runs/form/CourseRunForm.tsx

```diff
@@ -58,16 +60,18 @@ export function CourseRunForm({ courseRun, ...props }: Props) {
 
   const getDefaultValues = () => {
     return {
-      title: courseRun?.title ?? "",
-      course: courseRun?.course ?? (null as any), // to not trigger type validation for the default values
-      resource_link: courseRun?.resource_link ?? "",
-      start: courseRun?.start ?? null,
-      end: courseRun?.end ?? null,
-      enrollment_start: courseRun?.enrollment_start ?? null,
-      enrollment_end: courseRun?.enrollment_end ?? null,
-      languages: selectLanguageUtils.getObjectsFromValues(courseRun?.languages),
-      is_gradable: courseRun?.is_gradable ?? false,
-      is_listed: courseRun?.is_listed ?? false,
+      title: defaultCourseRun?.title ?? "",
+      course: defaultCourseRun?.course ?? (null as any), // to not trigger type validation for the default values
```

Comment:
Did you try to update the RegisterSchema instead to mark course as nullable ?
If all working like I understand, it will accept null as value, but on submit it should trigger an error value is missing.

```diff
-    course: Yup.object<Course>().required(),
+    course: Yup.object<Course>().nullable().required(),
```